pkg/gofr/datasource/solr/solr.go

```diff
@@ -153,70 +151,70 @@ func (c *Client) ListFields(ctx context.Context, collection string, params map[s
-// params can be used to specify the format of response
+// params can be used to specify the format of response.
 func (c *Client) Retrieve(ctx context.Context, collection string, params map[string]any) (any, error) {
 	url := c.url + "/" + collection + "/schema"
 	startTime := time.Now()
 
-	resp, err, span := c.call(ctx, http.MethodGet, url, params, nil)
+	resp, span, err := c.call(ctx, http.MethodGet, url, params, nil)
 
-	c.sendOperationStats(ctx, &QueryLog{Type: "Retrieve", Url: url}, startTime, "retrieve", span)
+	c.sendOperationStats(ctx, &QueryLog{Type: "Retrieve", URL: url}, startTime, "retrieve", span)
 
 	return resp, err
 }
 
-// AddField adds Field in the schema for the specified collection
+// AddField adds Field in the schema for the specified collection.
 func (c *Client) AddField(ctx context.Context, collection string, document *bytes.Buffer) (any, error) {
 	url := c.url + "/" + collection + "/schema"
 	startTime := time.Now()
 
-	resp, err, span := c.call(ctx, http.MethodPost, url, nil, document)
+	resp, span, err := c.call(ctx, http.MethodPost, url, nil, document)
 
-	c.sendOperationStats(ctx, &QueryLog{Type: "AddField", Url: url}, startTime, "add-field", span)
+	c.sendOperationStats(ctx, &QueryLog{Type: "AddField", URL: url}, startTime, "add-field", span)
 
 	return resp, err
 }
 
-// UpdateField updates the field definitions in the schema for the specified collection
+// UpdateField updates the field definitions in the schema for the specified collection.
 func (c *Client) UpdateField(ctx context.Context, collection string, document *bytes.Buffer) (any, error) {
 	url := c.url + "/" + collection + "/schema"
 	startTime := time.Now()
 
-	resp, err, span := c.call(ctx, http.MethodPost, url, nil, document)
+	resp, span, err := c.call(ctx, http.MethodPost, url, nil, document)
 
-	c.sendOperationStats(ctx, &QueryLog{Type: "UpdateField", Url: url}, startTime, "update-field", span)
+	c.sendOperationStats(ctx, &QueryLog{Type: "UpdateField", URL: url}, startTime, "update-field", span)
 
 	return resp, err
 }
 
-// DeleteField deletes the field definitions in the schema for the specified collection
+// DeleteField deletes the field definitions in the schema for the specified collection.
 func (c *Client) DeleteField(ctx context.Context, collection string, document *bytes.Buffer) (any, error) {
 	url := c.url + "/" + collection + "/schema"
 	startTime := time.Now()
 
-	resp, err, span := c.call(ctx, http.MethodPost, url, nil, document)
+	resp, span, err := c.call(ctx, http.MethodPost, url, nil, document)
 
-	c.sendOperationStats(ctx, &QueryLog{Type: "DeleteField", Url: url}, startTime, "delete-field", span)
+	c.sendOperationStats(ctx, &QueryLog{Type: "DeleteField", URL: url}, startTime, "delete-field", span)
 
 	return resp, err
 }
 
-// Response stores the response from SOLR
+// Response stores the response from SOLR.
```

Comment:
```suggestion
// Response stores the response from Solr.
```

Solr is not an acronym, but a brand, written Solr